docs/learn/avalanche/subnets-overview.md

```diff
@@ -2,107 +2,92 @@
+can host many blockchains.
 
-Avalanche's 3 built-in blockchains: Platform Chain (P-Chain), Contract Chain
-(C-Chain) and Exchange Chain (X-Chain) are validated and secured by all the
-Avalanche validators which compose a special Subnet and is referred as the
-Primary Network.
+Avalanche's [Primary Network](avalanche-platform.md) is a special Subnet running three blockchains:
+
+- The Platform Chain (P-Chain)
+- The Contract Chain (C-Chain)
+- The Exchange Chain (X-Chain)
 
-By definition, all Subnet validators must also validate the Avalanche Primary Network.
 
 ![image](/img/subnet-validators.png)
 
 (Image adopted from [this article](https://www.coinbase.com/cloud/discover/dev-foundations/intro-to-avalanche-subnets))
 
-Subnets are independent, they specify their own execution logic, determine their
-own fee regime, maintain their own state, facilitate their own networking, and
-provide their own security. They don’t share execution thread, storage[^1] or
-networking with other Subnets including the Primary Network, effectively
-allowing the network to scale up easily while enabling lower latency, higher
-transactions per second (TPS), and lower transaction costs provided by the
-Avalanche Consensus.
+:::info
+Every validator, regardless of Subnet involvement,
+**must** validate the Primary Network.
+:::
 
 ## Advantages
 
-A Subnet manages its own membership, it can create its own token economics and
-rules, and may require that its constituent validators have certain properties.
-
-### Independent Token Economics
-
-Subnets can have their own token economics with their own native tokens and fee
-markets. They can launch their own blockchains with customized virtual machines.
-See [Customize a Subnet](/docs/subnets/customize-a-subnet.md) for more details.
+### Independent Networks
 
-### Compliance
+- Subnets specify their own execution logic, determine their
+own fee regime, maintain their own state, facilitate their own networking, and
+provide their own security. 
+- Each Subnet's performance is isolated from other Subnets in the ecosystem, so increased usage on
+one Subnet won't affect another.
+- Subnets can have their own token economics with their own native tokens, fee
+markets, and incentives determined by the Subnet deployer. 
+- One Subnet can host multiple blockchains with customized [virtual machines](virtual-machines.md).
+- They don’t share execution thread, storage[^1], or
+networking with any other Subnets, including the Primary Network.
 
-Avalanche’s Subnet architecture makes regulatory compliance manageable. As
-mentioned above, a Subnet may require validators to meet a set of requirements.
+### Native Interoperability
 
-Some examples of requirements include:
+- Avalanche Warp Messaging enables native cross-Subnet communication and allows Virtual Machine (VM)
+developers to implement arbitrary communication protocols between any two Subnets.
+<!-- - would love a bullet about native bridges -->
```

Comment:
Remove this comment if it's not needed and capture it in a github issue if it's something which we want to follow up on in a later PR.